src/main/java/org/folio/innreach/external/client/feign/InventoryStorageClient.java

```diff
@@ -0,0 +1,13 @@
+package org.folio.innreach.external.client.feign;
+
+import org.folio.innreach.external.dto.InstanceIterationRequest;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+
+@FeignClient("inventory-storage")
+public interface InventoryStorageClient {
+
+  @PostMapping("/instance-storage/instances/iteration")
+  void startInitialContribution(@RequestBody InstanceIterationRequest request);
+}
```

Comment:
It seems like `InventoryStorageClient` and `InventoryClient` are both clients of `mod-inventory-storage`. Can we use existing client for this endpoint?

Reply:
Ignore the comment above, these are different modules